src/main/features/dashboard/views/macro/claimStatus/paidInFullDefendant.njk

```diff
@@ -11,8 +11,10 @@
 
 {% macro paidInFullDefendantClaimDetails(claim) %}
   <h2 class="heading-small">{{ t('This claim is settled') }}</h2>
-  <p>{{ t('{{ claimantName }} confirmed you paid on {{ moneyReceivedOn }}.',
-      { claimantName: claim.claimData.claimant.name, moneyReceivedOn: claim.moneyReceivedOn | date }) }}</p>
+  <p>{{ t('{{ claimantName }} confirmed you paid {{ totalAmountTillToday }} on {{ moneyReceivedOn }}.',
+      { claimantName: claim.claimData.claimant.name,
+        totalAmountTillToday: claim.totalAmountTillToday | numeral,
```

Comment:
This value will change day to day if interest is included. we need to either save the value at the time the case is settled or change the content

Reply:
content agreed with business and updated. we are not showing amount for now. I believe we are raising bug later - @nsonje is handling